connaisseur/tests/test_alert.py

```diff
@@ -2,13 +2,8 @@
 from datetime import datetime, timedelta
 import json
 
-from connaisseur.alert import (
-    Alert,
-    send_alerts,
-    call_alerting_on_request,
-    get_alert_config_validation_schema,
-    load_config,
-)
+import conftest as fix
+import connaisseur.alert as al
```

Comment:
Is there a reason for  not importing the functions themselves? The prefix `al` doesn't really tell anything so if we don't want to import the functions/classes themselves I'd go for plain `alert` instead for better readability.